### PR TITLE
Added a nice message For UIP to show on the main screen

### PR DESCRIPTION
**Pull request by vishvanath45**

### CODE REVIEW

File: UIP.py [hunk 1]
```diff
@@ -1,6 +1,12 @@
 from scheduler import scheduler
 
+print ("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
+
 if __name__ == "__main__":
+<<<<<<< 2185a6f5ba43b272c70dd78f1edf0696934debb4
```
Comment (abhsag24) [inline UIP.py]: this shouldn't be there

File: UIP.py [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+print ("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
+
 if __name__ == "__main__":
+<<<<<<< 2185a6f5ba43b272c70dd78f1edf0696934debb4
   print("Press 1 to connect to internet or any other key to use this application in offline mode")
   offline = int(input()) != 1
   scheduler(offline)
+=======
+	scheduler()
+>>>>>>> Added a nice message For UIP to show on the main screen
```
Comment (abhsag24) [inline UIP.py]: remove these they aren't a part of your code

File: UIP.py [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
 
+print ("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
+
 if __name__ == "__main__":
+<<<<<<< 2185a6f5ba43b272c70dd78f1edf0696934debb4
   print("Press 1 to connect to internet or any other key to use this application in offline mode")
   offline = int(input()) != 1
   scheduler(offline)
+=======
+	scheduler()
```
Comment (abhsag24) [inline UIP.py]: similarly this

File: UIP.py [hunk 4]
```diff
@@ -1,6 +1,12 @@
 from scheduler import scheduler
 
+print ("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
+
 if __name__ == "__main__":
+<<<<<<< 2185a6f5ba43b272c70dd78f1edf0696934debb4
   print("Press 1 to connect to internet or any other key to use this application in offline mode")
   offline = int(input()) != 1
   scheduler(offline)
+=======
```
Comment (abhsag24) [inline UIP.py]: and this

File: UIP.py [hunk 5]
```diff
@@ -1,6 +1,12 @@
 from scheduler import scheduler
 
+print ("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
```
Comment (abhsag24) [inline UIP.py]: move this print statement inside `if __name__ == "__main__":`

**Comment by vishvanath45** [comment]
Did changes accordingly.

File: UIP.py [hunk 6]
```diff
@@ -1,12 +1,8 @@
...
-  print("Press 1 to connect to internet or any other key to use this application in offline mode")
-  offline = int(input()) != 1
-  scheduler(offline)
-=======
-	scheduler()
->>>>>>> Added a nice message For UIP to show on the main screen
+      print ("Hey this is UIP! you can use it to download images from reddit 	and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")
+      offline = int(input())!=1
+      scheduler(offline)
+      scheduler()
```
Comment (abhsag24) [inline UIP.py]: we dont need this remove this please, also always test code by running `python3 UIP.py --force` before push :P

**Comment by vishvanath45** [comment]
i have merged my previous commits into single commit. please review.

**Comment by abhsag24** [comment]
@vishvanath45 Hey awesome work everything is exactly as it is supposed to be, the only issue is now you've got a merge conflict. If you want to learn how to resolve merge conflicts just ping us on gitter :D

**Comment by vishvanath45** [comment]
Resolved conflicts.

**Comment by vishvanath45** [comment]
Successfully merged my last 2 commits into one.

File: UIP.py [hunk 7]
```diff
@@ -1,10 +1,12 @@
 from scheduler import scheduler
 
 if __name__ == "__main__":
+
   try:
-    print("Press 1 to connect to internet or any other key to use this application in offline mode")
-    offline = int(input()) != 1
+    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")
```
Comment (nkprince007) [inline UIP.py]: This is good enough, but generally it's preferrable to use two print statements rather than writing oneliner, or you could just use python multiline string with
```
print(''' Hello\n
This is a multiline string\n''');
```

Comment (vishvanath45) [inline UIP.py]: Thanks for your suggestion. :D

Comment (abhsag24) [inline UIP.py]: @vishvanath45 i would agree with @nkprince007  to use two print statement and i would also suggest to move your print statement above the `try` statement.

**Comment by abhsag24** [comment]
@vishvanath45 i think you just need to make this slight change and use `git commit --amend` if you face any problem in that. You can always ping me or @nkprince007 or @DarkSouL11 on gitter :P

File: UIP.py [hunk 8]
```diff
@@ -1,12 +1,11 @@
...
+    
 if __name__ == "__main__":
-
-  try:
-    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")
-    offline = int(input())!=1
+    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
+try:
+    print("Press 1 to connect to internet or any other key to use this application in offline mode")
+    offline=int(input())!=1
```
Comment (abhsag24) [inline UIP.py]: unnecessary change though i'll ignore it for now

File: UIP.py [hunk 9]
```diff
@@ -1,12 +1,11 @@
 from scheduler import scheduler
-
+
```
Comment (abhsag24) [inline UIP.py]: unnecesssary still ignored :P

File: UIP.py [hunk 10]
```diff
@@ -1,12 +1,11 @@
 from scheduler import scheduler
-
+    
 if __name__ == "__main__":
-
-  try:
-    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")
-    offline = int(input())!=1
+    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.")
+try:
```
Comment (abhsag24) [inline UIP.py]: why unindent? will no longer work :(

**Comment by abhsag24** [comment]
@vishvanath45 hey now you have 3 commits  can you rebase /merge them to have just 1 commit and also do away with those unnecessary changes? like the unindentation of the try catch block:(

File: UIP.py (hunk 9 above)
Comment (nkprince007) [inline UIP.py]: @vishvanath45 I think it's better if you use a text editor that shows up git diff in its UI. So that you avoid unnecessary changes in files, and do leave blank lines as is. My preferred text editor is atom, developed by github.

**Comment by abhsag24** [comment]
i think it's mergeable :D finally
